starter-code/apple-pie-starter-code/index.html

```diff
@@ -1,35 +1,91 @@
-
-Ingredients
-1 recipe pastry for a 9 inch double crust pie
-1/2 cup unsalted butter
-3 tablespoons all-purpose flour
-1/4 cup water
-3 tablespoons all-purpose flour
-1/4 cup water
-1/2 cup white sugar
-1/2 cup packed brown sugar
-8 Granny Smith apples - peeled, cored and sliced
-
-Directions
-Preheat oven to 425 degrees F (220 degrees C). Melt the butter in a saucepan. Stir in
-flour to form a paste. Add water, white sugar and brown sugar, and bring to a boil. Reduce
-temperature and let simmer.
-
-Place the bottom crust in your pan. Fill with apples, mounded slightly. Cover with a
-lattice work crust. Gently pour the sugar and butter liquid over the crust. Pour slowly so
-that it does not run off.
-
-Bake 15 minutes in the preheated oven. Reduce the temperature to 350 degrees F (175
-degrees C). Continue baking for 35 to 45 minutes, until apples are soft.
+
+  <header class='header'>
+      <h1>Apple Pie</h1>
+  </header>
+  <div>
+    <section>
+      <article class="text">
+        <h3> This was my grandmother's apple pie recipe. I have never seen another one
+          quite like it. It will always be my favorite and has won me several first place
+          prizes in local competitions. I hope it becomes one of your favorites as well!</h3>
+      </article>
+    </section>
+  </div>
+
+  <div class="image">
+
+    <img src='./images/recipe-info.png'>
+
+    
+
+  </div>
+
+  <div>
+
+    <article class="title">
+
+      <hr>
+
+      <h2 >Ingredients</h2>
+
+    </article>
+
+    <article class="text">
+      <ul>
+        <li>1 recipe pastry for a 9 inch double crust pie</li>
+        <li>1/2 cup unsalted butter</li>
+        <li>3 tablespoons all-purpose flour</li>
+        <li>1/4 cup water</li>
+        <li>3 tablespoons all-purpose flour</li>
+        <li>1/4 cup water</li>
+        <li>1/2 cup white sugar</li>
+        <li>1/2 cup packed brown sugar</li>
+        <li>8 Granny Smith apples - peeled, cored and sliced</li>
+      </ul>
+    </article>
+    <article>
+  </div>
+<div class="image">
+
+    <img src='./images/cooking-info.png'>
+
+  
+
+</div>
+
+<div>
+    <article class="title">
+
+      <hr>
+
+      <h2>Directions</h2>
+
+    </article>
+
+    <article class="text">
+
+      <ol>
+        <li>Preheat oven to 425 degrees F (220 degrees C). Melt the butter in a saucepan. Stir in
+          flour to form a paste. Add water, white sugar and brown sugar, and bring to a boil. Reduce
+          temperature and let simmer.</li>
+
+        <li>Place the bottom crust in your pan. Fill with apples, mounded slightly. Cover with a
+          lattice work crust. Gently pour the sugar and butter liquid over the crust. Pour slowly so
+          that it does not run off.</li>
+
+        <li>Bake 15 minutes in the preheated oven. Reduce the temperature to 350 degrees F (175
+          degrees C). Continue baking for 35 to 45 minutes, until apples are soft.</li>
+      </ol>
+    </article>
+</div>
```

Comment:
You could have wrapped this whole part into a div or section to which you would have applied the padding-left and right properties to create equal space around all your elements.